src/x86/ffi64.c

```diff
@@ -350,7 +350,7 @@ examine_argument (ffi_type *type, enum x86_64_reg_class classes[MAX_CLASSES],
 		  _Bool in_return, int *pngpr, int *pnsse)
 {
   size_t n;
-  int i, ngpr, nsse;
+  unsigned int i, ngpr, nsse;
```

Comment:
This changes the type of ngpr and nsse, but those are later assigned like `*pngpr = ngpr;` -- where pngpr is a pointer-to-int.